MathLib/mechanics_invariants.h

```diff
@@ -32,89 +32,87 @@ struct Invariants final
 
     /// Determinant of a matrix in Kelvin vector representation.
     static double determinant(
-        Eigen::Matrix<double, KelvinVectorSize, 1> const& vec);
+        Eigen::Matrix<double, KelvinVectorSize, 1> const& v);
 
     /// Inverse of a matrix in Kelvin vector representation.
     static Eigen::Matrix<double, KelvinVectorSize, 1> inverse(
-        Eigen::Matrix<double, KelvinVectorSize, 1> const& vec);
+        Eigen::Matrix<double, KelvinVectorSize, 1> const& v);
 
     static double effectiveStress(
-        Eigen::Matrix<double, KelvinVectorSize, 1> const& deviatoric_stress)
+        Eigen::Matrix<double, KelvinVectorSize, 1> const& v)
```

Comment:
here `deviatoric_stress` would be appropriate because `effectiveStress` is a stress-related method.